data/json/items/generic.json

```diff
@@ -1032,6 +1032,51 @@
     "to_hit": -3,
     "flags": [ "TRADER_AVOID", "NO_REPAIR" ]
   },
+  {
+    "type": "GENERIC",
+    "id": "broken_mech_recon",
+    "name": "broken mech recon",
```

Comment:
Maybe rename it to `broken recon mech`, like the combat mech variant below? The same for lifter mech.